### ISSUE TITLE
StyleCop warnings not solved.

### ISSUE THREAD

**Issue by Paresh-Solanki** [codecadwallader/codemaid#484]
### Environment
- Visual Studio Professional 2015
- CodeMaid version: 10.4
- Code language:  C#

### Description
CodeMaid does not solve xml summary warnings of stylecop.

### Current behavior
        /// <summary>
        /// Get Cashier Details Entities
        /// </summary>
        /// <param name="storeNumber">store Number</param>
        /// <returns>List of cashier details entities</returns>
        public CashierDetailsResponseModel GetCashierDetails(string storeNumber)
        {
            var cashierDetails = _tillPointConversationRepositoryManager.GetCashierDetails(storeNumber);

            return _tillPointConversationProcessor.CreateCashierDetailsModel(cashierDetails);
        }

NOTE: If I remove param element, or I add extra lines in between the xml documentation, or add extra line between xml and method, codemaid will not solve that.

In short it does not solves SA1506, SA1507, SA1611 - related to XML documentation.

### Expected behavior
It should remove extra lines and at least add missing elements of xml.

**Comment by codecadwallader** [comment]
Thank you for the suggestion.

**Comment by Paresh-Solanki** [comment]
Hello, any workaround for this as of now?

**Comment by codecadwallader** [comment]
You may want to look into StyleCop directly since it's their rules you're referencing:  https://github.com/DotNetAnalyzers/StyleCopAnalyzers
